tcmur_cmd_handler.c

```diff
@@ -1689,6 +1689,18 @@ static int handle_caw_check(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	uint64_t start_lba = tcmu_get_lba(cmd->cdb);
 	uint8_t sectors = cmd->cdb[13];
 
+	/* From sbc4r12a section 5.3 COMPARE AND WRITE command
+	 * If the number of logical blocks exceeds the value in the
+	 * MAXIMUM COMPARE AND WRITE LENGTH field(see 6.64 block limits VPD page)
+	 * then the device server shall terminate the command with CHECK CONDITION
+	 * status with the sense key set to ILLEGAL REQUEST and the additional sense
+	 * code set to INVALID FIELD IN CDB.
+	 */
+	if (sectors > MAX_CAW_LENGTH) {
+		tcmu_dev_err(dev, "Received caw length %u greater than max caw length %u.\n",
+					 sectors, MAX_CAW_LENGTH);
```

Comment:
When you start a new line with the variables align it so it starts under the 'd' in "dev".

I fixed it for you and hand merged it here
https://github.com/open-iscsi/tcmu-runner/commit/501b9d9ef91ab0aff65d13d0d02be91c7a8fc74c